cmake/test_util.cmake

```
@@ -30,6 +30,11 @@ function(create_standard_test)
     target_compile_definitions(test_${TEST_NAME} PUBLIC USE_JSON)
   endif()
 
+  if(ENABLE_LLVM)
+    target_link_libraries(test_${TEST_NAME} PRIVATE ${llvm_libs})
+    target_compile_definitions(test_${TEST_NAME} PUBLIC USE_LLVM)
+  endif()
+
```

```suggestion
  endif()

  if(ENABLE_LLVM)
    target_link_libraries(test_${TEST_NAME} PRIVATE ${llvm_libs})
  endif()

```

Do we even need these compile definitions?

---

good point, I don't think we do.